fmriprep/workflows/base.py

```diff
@@ -147,7 +148,7 @@ def init_single_subject_wf(subject_id):
                 subject_id, task_id if task_id else '<all>')
         )
 
-    if not subject_data['t1w']:
+    if not anat_derivatives and not subject_data['t1w']:
```

Comment:
This check should go after the branch in line 220 because you can imagine the expectations are not met and the T1w is not provided either.